project/src/molecules/system_builder.py

```diff
@@ -1,8 +1,9 @@
 """Methods used to create systems from job statepoint."""
 import mbuild as mb
 from mbuild.lib.molecules.water import WaterSPC
-from methane_ua import MethaneUA
-from pentane_ua import PentaneUA
+
+from project.src.molecules.methane_ua import MethaneUA
```

Comment:
I get this error

ModuleNotFoundError: No module named 'project.src'; 'project' is not a package

Reply:
This works for me:

```python
from methane_ua import MethaneUA
from pentane_ua import PentaneUA
```

Reply:
Where were you running this from @ramanishsingh ? This should be ran inside the `project` folder.

```bash
conda activate mosdef-study38
cd reproducibility_study/project
python src/engines/mcccs/project.py
```

At least I am pretty sure that is the case

Reply:
Is it working for you?
I am still getting project is not a package error.

Reply:
I changed both relative statement, waiting for tests.

Reply:
I revert to the original import statement since `from pentane_ua import PentaneUA` would not pass the CI tests

Reply:
I think this import issue has something to do with the library is named `project` get imported in a directory that also has a `project.py`. I am not sure if there is an easy fix to this, but we may consider changing the library name (maybe `reproducibility_project`)? @justinGilmer

Reply:
Ahh. Yeah that might be it @daico007     If you want to try changing the library name to reproducibility_project, let’s see if that fixes it

Reply:
@justinGilmer I changed the name of the main dir to be `reproducibility_project`. @ramanishsingh can you try and see if this works for you.

Reply:
This worked for me after I added this in my `project.py`

```python
sys.path.append(Project().root_directory() + "/..")
```